policyengine_us/parameters/gov/states/ny/tax/income/supplemental/incremental_benefit/widow.yaml

```diff
@@ -5,33 +5,39 @@ metadata:
   amount_unit: currency-USD
```

Comment:
inflation?

Reply:
is uprating needed for all parameters? wasn't sure how to distinguish as legal code didn't mention any "inflation" language

Reply:
Does it talk about "consumer price" or does it address adjustments based on another legal code section?

Reply:
No, it does not